Plugins/IMA/IMAPlugin.swift

```diff
@@ -248,6 +233,7 @@ extension PKErrorDomain {
     private func updateAdTag(_ adTag: String, tagTimeKeyForRemove: TimeInterval) {
         self.tagsTimes![tagTimeKeyForRemove] = nil
         self.destroyManager()
+        self.contentComplete()
```

Comment:
This is for manual ad tags list only.

When requesting ads from the same URL in a short time you can receive error for ad could not been loaded and this fixes it. 
If you think this should never happen I will remove this.